addons/auth_signup/data/auth_signup_data.xml

```diff
@@ -26,7 +26,7 @@
                         ${object.name}
                     </span>
                 </td><td valign="middle" align="right">
-                    <img src="/logo.png?company=${object.company_id.id}" style="padding: 0px; margin: 0px; height: auto; width: 80px;" alt="${object.company_id.name}"/>
+                    <img src="/logo.png?company=${object.company_id.id}&amp;dbname=${object.env.cr.dbname}" style="padding: 0px; margin: 0px; height: auto; width: 80px;" alt="${object.company_id.name}"/>
```

Comment:
@tivisse 

we use "company" parameter in company_logo controller. https://github.com/odoo/odoo/blob/master/addons/web/controllers/main.py#L1185

I think we have to use "cids" parameter for a company in every url.

can we rename "company" with "cids" in company_logo controller?

cc @tde-banana-odoo